NMS-12373: Provisiond Docs Update (#3521)
OpenNMS/opennms: docs/modules/operation/pages/provisioning/snmp-profile.adoc

1. `profile1` doesn't have a filter expression.
This profile will be tried for every interface.

> 2. `profile2` has a filter expression that compares `iphostname` (the hostname resulting from a reverse DNS lookup of the IP address being fitted) against a pre-configured value.

Reviewer: "preconfigured" is actually standard use according to Oxford and Webster's.

Author: ```suggestion
2. `profile2` has a filter expression that compares `iphostname` (the hostname resulting from a reverse DNS lookup of the IP address being fitted) against a preconfigured value.
```

This removes IP address `172.1.0.255` at location `MINION` from the system-wide SNMP configuration so that this IP address can be fitted to a new profile.
This command might be useful when an IP address formerly assigned to an SNMPv2c-capable switch is reassigned to an SNMPv3-capable load balancer.

== Using SNMP profiles with the SNMP detector

> By default SnmpDetector doesn't use SNMP profiles. Add property `useSnmpProfiles` and set it to `true` in order to use SNMP Profiles.

Reviewer: ```suggestion
By default, SnmpDetector doesn't use SNMP profiles. 
Add property `useSnmpProfiles` and set it to `true` to use SNMP profiles.
```

== Using SNMP profiles with the SNMP detector

> By default, SnmpDetector doesn't use SNMP profiles. 

Reviewer: ```suggestion
By default, SnmpDetector does not use SNMP profiles. 
```
